Makefile.pre.in

```diff
@@ -1587,14 +1587,15 @@ regen-cases:
 	$(PYTHON_FOR_REGEN) \
 	    $(srcdir)/Tools/cases_generator/generate_cases.py \
 		$(CASESFLAG) \
-		-o $(srcdir)/Python/generated_cases.c.h.new \
```

Comment:
I'd put this line back; the default always overwrites Python/generated_cases.c.h.

Reply:
I've change `generate_cases.py` to no longer generate the tier 1 code.